src/metabase/api/card.clj

```diff
@@ -1155,16 +1155,50 @@ saved later when it is ready."
        :num-columns (count (first rows))
        :num-rows    (count (rest rows))})))
 
+(defn- can-upload-error
+  "Returns an ExceptionInfo object that the user can upload to the given database and schema. Throws an error if not."
```

Comment:
This docstring seems incorrect?

Reply:
Fixed [Fix docstring](https://github.com/metabase/metabase/pull/33990/commits/7d082a57b88f2516346dd6c6164e1f4b82fe5302)